src/Command/NamespaceToPSR4Command.php

```diff
@@ -49,7 +49,8 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $path = (string) $input->getArgument('path');
-        $namespaceRoot = (string) $input->getOption('namespace-root');
+        $namespaceRoot = rtrim((string) $input->getOption('namespace-root'), '\\');
```

Comment:
This fixes the issue where an incorrect `App\\\Controller` gets generated on Windows.